config/prometheus/cert-manager-monitor.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: cert-manager
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/instance: cert-manager
+    app.kubernetes.io/name: cert-manager
+    app.kubernetes.io/version: v1.9.1
```

Comment:
@thejasn Could you please confirm if we need this `app.kubernetes.io/version` label here?